### DISCUSSION TITLE
Dúvidas sobre abertura e fechamento do caixa.

### DISCUSSION THREAD

**Discussion by lucassl2020** [EuricoDNJR/SG-VILLA-DOLCE#89] [Q&A] [answered]
Existe uma quantidade limite de caixas por dia? Por exemplo, o caixa só pode ser aberto e fechado 3 vezes no dia.

Existe quantidade mínima de tempo após o caixa ser aberto para poder fecha-lo?

**Comment by EuricoDNJR** [comment, marked as answer]
- Como o estabelecimento só funciona no período de 17:00 - 22:00, é necessário de apenas 1 caixa por dia, o mesmo sendo aberto no início das operações do dia e fechado no fim.
- Não existe quantidade mínima de tempo.
